site/content/contribute/mobile/build-your-own/assets-white-label.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Assets & White Labeling"
+date: 2018-05-20T11:35:32-04:00
+weight: 3
+subsection: Build Your Own App
+---
+
+We've made it easy to white label the mobile app and to replace and override the assets used, however, you have to [Build Your Own App](/contribute/mobile/build-your-own) from Source\_.
```

Comment:
There is an underline at the end of the sentence that I don't know if it belongs here